pkg/crlprovider/crl_fetcher.go

```diff
@@ -0,0 +1,75 @@
+/*
+Copyright The Ratify Authors.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package crlprovider
+
+import (
+	"crypto/x509"
+	"fmt"
+
+	corecrl "github.com/notaryproject/notation-core-go/revocation/crl"
```

Comment:
This CRL provider implementation depends on the notary project for fetching and caching.
From my point of view, the cached CRL can be reused in multi-verifier with some update in the interface. 
Besides, CRL has a closer connection with certificates and its management rather than the verifying process that is controlled by the verifier.

Add this discussion to PR review session for more feedback. (❁´◡`❁)

Reply:
On second thought, you’re right. Let’s keep it simple and clean.